bentoml/_internal/bento/bento.py

```diff
@@ -288,8 +315,77 @@ def export(self, path: str):
         out_fs = fs.open_fs(path, create=True, writeable=True)
         fs.mirror.mirror(self._fs, out_fs, copy_if_newer=False)
 
-    def push(self):
-        pass
+    @inject
+    def push(self, *, force: bool = False, model_store: "ModelStore" = Provide[BentoMLContainer.model_store]):
```

Comment:
Could we put bento push/pull and models push/pull in yatai client class?  

* yatai_rest_api_client: low-level API, methods directly mapping to REST APIs
* yatai_client: high-level user facing APIs: push_bento, pull_bento, push_model, pull_bento, login, swith_context,